.travis.yml

```diff
@@ -28,6 +28,7 @@ script:
   - export NVIM_NODE_LOG_LEVEL=debug
   - yarn
   - yarn link
+  - npm install -g ./
```

Comment:
Thank you for your fix. I wonder why this fixes CI because `yarn link` links this repository to the global. Could you please explain the reason?

Reply:
`npm link` adds the `neovim` module in the folder output of `npm root -g`.
`yarn link` doesn't.
`provider#node#Detect()` on neovim HEAD directly checks if `$(npm root -g)/neovim/bin/cli.js` exists.

Reply:
Thank you for your explanation. I think it should be improved. But it is not a scope of this PR.